site/_docs/reference.md

```diff
@@ -2794,6 +2794,7 @@ In the following:
 | s | MAP()                                          | Returns an empty map
 | s | MAP(key, value [, key, value]*)                | Returns a map with the given *key*/*value* pairs
 | s | MAP_CONCAT(map [, map]*)                       | Concatenates one or more maps. If any input argument is `NULL` the function returns `NULL`. Note that calcite is using the LAST_WIN strategy
+| s | MAP_CONTAINS_KEY(map, key)                     | Returns true if the *map* contains the *key*
```

Comment:
What does it return if the map does not contain the key? False? Null? Error? Maybe `Returns whether map contains key* is better because it has less ambiguity.

Reply:
done